index.html

```diff
@@ -6,15 +6,66 @@
     <title>Quiz App</title>
     <link rel="stylesheet" href="styles/index.css" />
   </head>
-  <body class="app-grid marginzero">
+  <body class="app-grid">
     <header class="app-grid__header">
       <h1>Quiz App</h1>
     </header>
 
-    <main class="app-grid__main"></main>
-    <section class="main__card">
-      <h2>Question</h2>
-    </section>
+    <main class="app-grid__main">
+      <section class="main__card">
+        <section class="card__question-section">
+          <h2>Question</h2>
+          <p>Which city is the capital of Germany?</p>
+        </section>
+        <section class="card__answer-section">
+          <details class="card__answer-toggle">
+            <summary>Show Answer</summary>
+            Germany's capital is Berlin.
+          </details>
+          <ul class="card__tag-selection">
+            <li class="card__tag">#Geography</li>
+            <li class="card__tag">#Germany</li>
+            <li class="card__tag">#Politics</li>
+          </ul>
+        </section>
+      </section>
+
+      <section class="main__card">
+        <section class="card__question-section">
+          <h2>Question</h2>
+          <p>Which city is the capital of the Netherlands?</p>
+        </section>
+        <section class="card__answer-section">
+          <details class="card__answer-toggle">
+            <summary>Show Answer</summary>
+            The dutch capital is Amsterdam.
+          </details>
+          <ul class="card__tag-selection">
+            <li class="card__tag">#Geography</li>
+            <li class="card__tag">#Netherlands</li>
+            <li class="card__tag">#Politics</li>
+          </ul>
+        </section>
+      </section>
+
+      <section class="main__card">
+        <section class="card__question-section">
+          <h2>Question</h2>
+          <p>Which city is the capital of Denmark?</p>
+        </section>
+        <section class="card__answer-section">
+          <details class="card__answer-toggle">
+            <summary>Show Answer</summary>
+            Denmark's capital is Copenhagen.
+          </details>
+          <ul class="card__tag-selection">
+            <li class="card__tag">#Geography</li>
+            <li class="card__tag">#Denmark</li>
+            <li class="card__tag">#Politics</li>
+          </ul>
+        </section>
+      </section>
+    </main>
     <nav class="app-grid__nav">
```

Comment:
Again, I would suggest this element being the block element of the navigation.
```suggestion
    <nav class="navigation">
```